lensterxyz/lenster
# feat: add enable DMs panel

## What does this PR do?

This PR adds a call to action to enable DMs. This will help onboard people into the network so they can receive DMs from their friends.

<img width="1456" alt="Screen Shot 2022-10-14 at 4 00 31 PM" src="https://user-images.githubusercontent.com/556051/195933200-b7dd3553-ad23-4b69-8a4a-25bd0ca5f3e8.png">

## Type of change




## How should this be tested?
- Create a new eth/polygon address
- Sign up with Lenster
- Notice the call to action to enable DMs on the homescreen
- Tapping it should take you to messages which will trigger a signature req
